modules/calc/src/main/java/com/opengamma/strata/calc/runner/CalculationFunction.java

```diff
@@ -103,6 +103,8 @@
    * Determines the market data required by this function to perform its calculations.
    * <p>
    * Any market data needed by the {@code calculate} method should be specified.
+   * <p>
+   * The set of measures may include measures that are not supported by this function.
```

Comment:
The docs of requirements() didn't specify the measures were limited, whereas the docs to calculate() did. Since doing so has an overhead, and code in requirements() usually doesn't care, it seemed like the best balance.